widgets/tokens/src/TokensTableHeader/CreateTokenModal.tsx

```diff
@@ -1,37 +1,64 @@
+import { isEqual } from 'lodash';
 import { useState } from 'react';
 import { flushSync } from 'react-dom';
 import { translationPath } from '../widget.consts';
 import { tableRefreshEvent } from '../TokensTable.consts';
 import { RequestStatus } from '../types';
 import CreatedToken from './CreatedToken';
-import type { ReceivedToken } from './CreateTokenModal.types';
+import type { ReceivedToken, TokensPostRequestBody } from './CreateTokenModal.types';
 
-const { useInput } = Stage.Hooks;
+const { useInput, useErrors } = Stage.Hooks;
 const { getT } = Stage.Utils;
-const { Modal, Icon, CancelButton, ApproveButton, Input, Form, LoadingOverlay, Message } = Stage.Basic;
+const { Modal, Icon, CancelButton, ApproveButton, DateInput, Input, Form, LoadingOverlay, Message } = Stage.Basic;
 
 const t = getT(`${translationPath}.createModal`);
+const expirationDateFormat = 'YYYY-MM-DD HH:mm';
+
+function getTokensPostRequestBody(description: string, expirationDate: string): TokensPostRequestBody {
+    const body: TokensPostRequestBody = {};
+    if (description) body.description = description;
+    if (expirationDate) body.expiration_date = moment.utc(moment(expirationDate)).format(expirationDateFormat);
+    return body;
+}
```

Comment:
We could extract 
> moment.utc(moment(expirationDate)).format(expirationDateFormat);

As a separate function, to make things more robust 🎉 

```suggestion
function formatRequestExpirationDate(expirationDate: string) {
    return moment.utc(moment(expirationDate)).format(expirationDateFormat);
}

function getTokensPostRequestBody(description: string, expirationDate: string): TokensPostRequestBody {
    const body: TokensPostRequestBody = {};
    if (description) body.description = description;
    if (expirationDate) body.expiration_date = formatRequestExpirationDate(expirationDate);
    return body;
}
```

Reply:
OK, will change.